spectree/plugins/starlette_plugin.py

```diff
@@ -22,7 +22,14 @@ def PydanticResponse(content):
     class _PydanticResponse(JSONResponse):
         def render(self, content) -> bytes:
             self._model_class = content.__class__
```

Comment:
The returned `List` class will be list, will this affect the correctness?

Reply:
No it is consistent with the existing implementation (i.e. if you return a list of serialized models,`_model_class` is set to `List`). We could on the other hand extract a bit more metadata and implement validation skipping, but I think it's a bit premature and could do in a followup MR.

Reply:
Sure.